src/slic3r/GUI/Plater.cpp

```diff
@@ -4055,8 +4055,6 @@ std::vector<size_t> Plater::priv::load_files(const std::vector<fs::path>& input_
                 q->skip_thumbnail_invalid = false;
                 return empty_result;
             }
-
-            model_object->ensure_on_bed(is_project_file);
```

Comment:
can we add a `if (!model_object->instances.empty())` check instead of deleting it?

Reply:
Yes, it might work. I'll check it.